core/commons/che-core-commons-tracing/src/main/java/org/eclipse/che/commons/tracing/TracingUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.eclipse.che.commons.tracing;
+
+import io.opentracing.Span;
+
+public final class TracingUtil {
+
+  public static void setErrorStatus(Span span, Throwable e) {
+    TracingTags.ERROR.set(span, true);
+    TracingTags.ERROR_REASON.set(span, e.getMessage());
+    TracingTags.SAMPLING_PRIORITY.set(span, 1);
+  }
+
+  public static void setWorkspaceIdAndMachineName(
```

Comment:
I think that `TracingUtil.setWorkspaceIdAndMachineName` says not much that tags will be set. Maybe `setMachineTag(String workspaceId, String machineName)` would be a bit better? I found it logical to put workspace id as a parameter even if the method name says nothing about it since a machine is always related to a workspace. But not sure if it will be explicit enough
Another thought: `TracingTags.setWorkspaceIdAndMachineName` or `TracingTags.setMachine` would look better IMHO

Reply:
I like idea with `TracingTags.setMachine`

Reply:
I've removed both for now and am just using `TracingTags` methods directly (it's one additional like but is clear about what we're doing). I can readd a `#setMachine` method if you think that is better though.

Reply:
I'm OK with removing `setMachine`